draft-ietf-rats-eat.md

```diff
@@ -219,32 +216,18 @@ claims.
 
 # Introduction
 
-Some of the goals and fundamentals in the security model for attestation are not the same as other security standards such as those aimed at privacy (e.g., TLS) and authentication (e.g. FIDO).
-The security model for attestation is not described here.
-Instead see {{RATS.Architecture}}.
+An Entity Attestation Token (EAT) is a message or token made up of claims about an entity.
+An entity may be a device, some hardware or some software.
+The claims are ultimately used by a relying party who decides if and how it will interact with the entity.
+The relying party may choose to trust, not trust or partially trust the entity.
```

Comment:
Wouldn't it be more precise to say that the relying party may choose to trust claims about an entity?

```suggestion
The relying party may choose to trust, not trust or partially trust claims about an entity.
```